Sydney Update

mattkelly4 opened the pull request:
- Heli ops
- ATIS updates
- Files consolidated into single YSSY file

chrisg06 commented on docs/aerodromes/sydney.md:
```diff
@@ -0,0 +1,272 @@
+
+### SY ADC / SY SMC
+#### Ground Operations
+As per this (link) diagram, **SY ADC** has Responsibility of the Runways, meaning **SY SMC** must always coordinate with **SY ADC** to allow aircraft to cross runways whilst taxiing. **SY SMC** may request, or **SY ADC** may elect, to release certain runways to the **SY SMC** controller, so they may let aircraft cross the runway without coordination (For example, Releasing runway 07/25 to **SY SM ...
```
missing link, or is it supposed to still be a placeholder?

mattkelly4 replied:
I didn't write this section, I assume it's a placeholder

chrisg06 commented on docs/terminal/Sydney TCU/operations.md:
```diff
@@ -87,20 +87,19 @@ Aim to vector aircraft to the following Intercept points during instrument PROPS
 !!! example
-    **SFW:** “BNZ444, Turn left heading 360, join final Runway 34L, from that heading Cleared Independent Visual Approach, report on final” `AIP GEN 3.4`   
+    **SFW:** “BNZ444, Turn left heading 360, join final Runway 34L, from that heading Cleared Independent Visual Approach” `AIP GEN 3.4`   
     **BNZ444:** "Left heading 360, join final Runway 34L, from that heading Cleared Independent Visual Approach, wilco, BNZ444"  
```
can remove wilco as report final was removed

mattkelly4 replied:
Fixed

mattkelly4 commented on docs/aerodromes/sydney.md:
```diff
@@ -0,0 +1,272 @@
+When SODPROPS are in operation, the ATIS OPR INFO shall include:  
+`SIMULTANEOUS OPPOSITE DIRECTION PARALLEL RUNWAY OPERATIONS IN PROGRESS`  
+
+When Independent Visual Approaches are run during PROPS, the OPR INFO must also include:
```
Good pickup, fixed

mattkelly4 commented:
Sorry last push for this one, tidied up everything now

zachbf commented:
@mattkelly4 only thing is that the ATIS should say "DO NOT PASS THRU ASSIGNED RWY CL" not "CENTRELINE" i think.
I've updated the code but the example image would need updating.
i've also fixed the merge conflicts by running `git merge origin/master`

mattkelly4 commented:
> @mattkelly4 only thing is that the ATIS should say "DO NOT PASS THRU ASSIGNED RWY CL" not "CENTRELINE" i think. I've updated the code but the example image would need updating. i've also fixed the merge conflicts by running `git merge origin/master`

Yeah perfect, I'll update the image quickly now.  Cheers
